### PR TITLE
notification bar enhanced

### PR DESCRIPTION
**Pull request by LalitChauhan27**
### Related Issue
- (1) Material UI for notification bar button class, (2) centre alignment for notif bar components #97

### Proposed Changes
- Material UI added for notification bar button class.
- Notification bar components now are in center.
- Added smooth collapse animation

### Checklist
- [x] Documentation of the code
- [x] Add your name to the CONTRIBUTORS file 
- [x] Testing your code in the local machine 

### Screenshots
P1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -92,9 +92,9 @@
     
     <!-- Dismissable notification bar  -->
     <div class="notif-bar-btn" id="close">
-        <div style="display: flex;width:73%;align-items: flex-end;">
+        <div style="display: flex;width:73%;align-items: flex-end; justify-content: center;"><!--To center only the text and the button-->
```
Comment (jay-io) [inline index.html:95]: Change `align-items: flex-end` to `align-items: center` to nicely centre the elements inside the notif-bar vertically.

File: index.html [hunk 2]
```diff
@@ -92,9 +92,9 @@
...
         <div style="margin-right: 1rem;">Join us on Discord!</div>
-        <a href="https://discord.gg/vYSspBkgNc"><Button style="background-color: rgb(54,66,86);color:white;margin-right: 1rem;border-radius: 0.5rem;">Join Discord</Button></a>
+        <a href="https://discord.gg/vYSspBkgNc"><button class="button-61" role="button">Join Discord</button></a><!--Material UI button added-->
     </div>
         <div style="display: flex;width: 10%;justify-content: flex-end; margin-right: 1rem;">
```
Comment (jay-io) [inline index.html:99]: Change style to `position:fixed; right: 30px`. That should be it.

**Review by jay-io — CHANGES_REQUESTED** [review]
@LalitChauhan27  You can now create a commit with the above changes. The PR will be updated automatically.

**Comment by LalitChauhan27** [comment]
Hey @jay-io, can you please add the 

label.

**Pull request merged**